Fix samples for AS policy rules (#1592)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

#### Condition properties

| Property     | Description                                                                                                                                                                                                                              | Type                            | Required for create or update |
| :----------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------ | :---------------------------- |
| clients      | For Policies, specifies which clients are included or excluded in the Policy                                                                                                                                                             | `include` and `exclude` lists   | True                          |

> | grantTypes   | Array of grantTypes this condition includes. Can include any of the following: `authorization_code`, `password`, `refresh_token`, or `client_credentials`. Determines the mechanism Okta uses to authorize the creation of the tokens.   | `include` list                  | True                          |

Reviewer: ```suggestion
| grantTypes   | Array of grantTypes that this condition includes. Accepted grantTypes: `authorization_code`, `password`, `refresh_token`, or `client_credentials`. Determines the mechanism Okta uses to authorize the creation of the tokens.   | `include` list                  | True                          |
```

Author: done

| people       | For rules, specifies which Users and Groups are included or excluded in the rule                                                                                                                                                         | `include` and `exclude` lists   | True                          |

> | scopes       | Array of Scopes this condition includes                                                                                                                                                                                                  | `include` list                  | True                          |

Reviewer: ```suggestion
| scopes       | Array of Scopes that this condition includes                                                                                                                                                                                                  | `include` list                  | True                          |
```

Author: done
